src/content/2024-10-08-gator-tank-competition.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: "/2024-gator-tank"
+date: "2024-10-09"
+title: "OSC Competes in and Wins UF Gator Tank 2024"
```

Comment:
On second though, can you please change this to "OSC Competes in UF Gator Tank 2024"